binding-service-impl/src/main/java/io/mosip/esignet/services/KeyBindingServiceImpl.java

```diff
@@ -54,12 +55,19 @@ public class KeyBindingServiceImpl implements KeyBindingService {
 	@Autowired
 	private KeyBindingHelperService keyBindingHelperService;
 
+	@Autowired
+	private CaptchaHelper captchaHelper;
+
+	@Value("${mosip.esignet.send-otp.attempts:3}")
```

Comment:
Is this property required @Md-Humair-KK?

Reply:
Yes @ase-101  - Actually captcha validation is part of CaptchaHelper class as you suggested earlier. Hence it is required.